### DISCUSSION TITLE
Color of the slider

### DISCUSSION THREAD

**Discussion by valentinavellani** [jspsych/jsPsych#1344] [Q&A]
Hi All,
in the slider plug in the default slider is blue (see the attached image).
This might bias subject's attention towards one side of the screen. 
I was wondering whether there was a way to modify the plug in to have the slider to be grey as the bar, or to be transparent.

Thank you in advance for your help!

![slider](https://user-images.githubusercontent.com/71888090/103660935-130fe780-4f66-11eb-8a8a-ee57f9ac7dc0.png)

**Comment by jodeleeuw** [comment]
Hi @valentinavellani ,

We added some custom CSS in 6.2 that should make the sliders appear more uniform across browsers. (The blue is the default styling applied by the browser, not something set by jsPsych.) Have you tried running your experiment with 6.2 instead?

Here's [the issue](https://github.com/jspsych/jsPsych/issues/695#issuecomment-726403540) where we discussed slider appearances, for reference.

**Reply by valentinavellani** [reply]
Thank @jodeleeuw for the suggestion.
I am using Gorilla, so I uploaded in resources the CSS 6.2 but I keep having the same issue. 
It seems to be specifically related to google chrome.

Do you have other ideas on how to fix this?

Thank you,
Valentina

**Reply by jodeleeuw** [reply]
I think you'll need to use the updated plugin from 6.2 in addition to the CSS file.

I don't know how custom CSS works on Gorilla. You could check in with them about upgrading to 6.2.
